### PR TITLE
Refactorings for v0.1.79

### PR DESCRIPTION
**Pull request by ingydotnet**

### CODE REVIEW

File: exercises/practice/bank-account/.meta/bank-account.ys [hunk 1]
```diff
@@ -32,19 +32,19 @@ defn- do-balance(op):
...
-  amount =: op.get-amount()
+  amount =: op:get-amount
   when amount > balance:
     die: 'amount must be less than balance'
   swap! account -: amount
 
 defn- get-amount(op):
   amount =: op.amount
-  if amount <= 0:
+  if amount < 0:
```
Comment (vendethiel) [inline exercises/practice/bank-account/.meta/bank-account.ys:48]: Why the change?

File: exercises/practice/protein-translation/.meta/protein-translation.ys [hunk 2]
```diff
@@ -4,7 +4,7 @@ defn proteins(strand):
   proteins =:
     strand.str('XX').partition(3)
           .map(join).map(translate-codon)
-          .take-while(\(_ != 'STOP'))
```
Comment (vendethiel) [inline exercises/practice/protein-translation/.meta/protein-translation.ys:7]: Can this use ne?

**Review by vendethiel — APPROVED** [review]

File: exercises/practice/bank-account/.meta/bank-account.ys (hunk 1 above)
Comment (ingydotnet) [reply exercises/practice/bank-account/.meta/bank-account.ys:48]: Because it worked the same, so I guess <= was wrong.
Not enough tests provided for that case I guess.

File: exercises/practice/protein-translation/.meta/protein-translation.ys (hunk 2 above)
Comment (ingydotnet) [reply exercises/practice/protein-translation/.meta/protein-translation.ys:7]: Yep. Nice! 69b5183
